index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Hendrid</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <a href="/" class="header-logo">
+            <h3>
+                Welcome
+            </h3>
+        </a>
+        <nav>
+            <img src="./assest/Union.svg" alt="menu link">
+        </nav>
+    </header>
+    <section class="headline">
+        <h2>
```

Comment:
- [ ] [REQUIRED] I think the main title (e.g., "I'm Marianne ...") is a h1 tag. In your case you have used a h2 tag. Please change that as it is part of a good SEO